data/random-teams.ts

```diff
@@ -2096,6 +2092,7 @@ export class RandomTeams {
 		const formes: string[][] = [];
 		for (const species of speciesPool) {
 			if (!(species.num in hasDexNumber)) continue;
+			if (requiredType && !species.types.includes(requiredType)) continue;
 			if (isNotCustom && (species.gen > this.gen ||
 				(species.isNonstandard && species.isNonstandard !== 'Unobtainable'))) continue;
 			if (requiredType && !species.types.includes(requiredType)) continue;
```

Comment:
you don't need this one, because I added this exact same line 2 lines down in another pr

Reply:
Woops, my bad. I started making changes having forgotten to pull first, and then remade everything still needed and clearly didn't check well enough. ty!